tha2/util.py

```diff
@@ -214,5 +214,4 @@ def convert_output_image_from_torch_to_numpy(output_image):
         numpy_image = grid_change_to_numpy_image(output_image, num_channels=4)
     else:
         raise RuntimeError("Unsupported # image channels: %d" % output_image.shape[0])
-    numpy_image = numpy.uint8(numpy.rint(numpy_image * 255.0))
```

Comment:
I've discovered a new problem: the Jupyter notebooks (tha2.ipynb and the other one) stopped working after this line is removed. Here's what it looks like: https://imgur.com/JPQwk2g

The code in the Jupyter notebook expects the output of this function to type dtype=uint8. Without this line, it becomes dtype=float32, which results in the image not being converted in the right way.

Please revert the removal of this line.